ViewPacker/tools/package.json

```diff
@@ -19,15 +19,15 @@
         "fs-extra": "^8.1.0",
         "glob": "^7.1.4",
         "mini-css-extract-plugin": "^0.8.2",
-        "node-sass": "^4.14.0",
-        "resolve-url-loader": "^3.1.0",
+        "node-sass": "^7.0.2",
+        "resolve-url-loader": "^3.1.4",
         "sass-loader": "^7.3.1",
         "thread-loader": "^2.1.3",
         "ts-loader": "^6.2.2",
         "typescript": "^4.2.2",
         "webpack": "^4.46.0",
         "webpack-cli": "^3.3.12",
-        "webpack-dev-server": "^3.10.3",
+        "webpack-dev-server": "^4.0.0",
```

Comment:
We need to check if the webpack config for the hot reload is still valid with this bump